src/shared/components/query/QueryStore.ts

```diff
@@ -1572,54 +1584,27 @@ export class QueryStore
 		this.initiallySelected.sampleListId = true;
 	}
 
-    @action setParamsFromLocalStorage()
+	// TODO: we should be able to merge this with the above since it accepts same interface
+    @action setParamsFromLocalStorage(legacySubmission:Partial<CancerStudyQueryUrlParams>)
     {
 
-        let legacySubmission: any = localStorage.getItem("legacyStudySubmission");
-
-        localStorage.removeItem("legacyStudySubmission");
-
-        if (legacySubmission) {
-
-            const parsedSubmission:any = JSON.parse(legacySubmission);

```

Comment:
how will all this info still be loaded into store?

Reply:
actually strike that.  in querystore the profiles will default.  in the results view, the query itself needs the params, but it gets them from url, not from querystore, make sense?

Reply:
ah okay, got it